scripts/omPrior.py

```diff
@@ -57,5 +57,3 @@
 omWetlandEmis.processEmissions(args.startDate, args.endDate)
 
 omOutputs.sumLayers()
-
-omPriorVerify.verifyEmis()
```

Comment:
I think that calling this is as part of the prior creation is still correct. The tests are in independent check that the emissions sum up correctly for the example domain.